doc/manual/customizing.rst

```diff
@@ -49,49 +89,202 @@ inherit them.  You can also manually copy the example file from
+  for the workspace type, whether that is a notebook tab, MDI window
+  titlebar, etc.
+
+Here is the standard layout, as an example:
+
+.. literalinclude:: ../../ginga/examples/layouts/standard/layout
+    :language: python
+
+In the above example, we define a top-level window consisting of a vbox
+(named "top") with 4 layers: a hbox ("menu"), hpanel ("hpnl"), a
+workspace ("toolbar") and another hbox ("status").  The main horizontal
+panel ("hpnl") has three containers: a workspace ("left"), a vbox
+("main") and a workspace ("right").  The "left" workspace is
+pre-populated with an "Info" tab containing a vertical panel of two
+workspaces: "uleft" and "lleft" (upper and lower left).  The "right"
+workspace is pre-populated with a "Dialogs" tab containing an empty
+workspace.  The "main" vbox is configured with four rows of workspaces:
+"channels", "cbar", "readout" and "operations". 
+
+.. note:: The workspace that has as a configuration option ``default:
+          True`` (in this example, "channels") will be used as the
+          default workspace where new channels should be created.
+
+
+.. _sec-pluginconfig:
+
+==============================
+Customizing the set of plugins
+==============================
+
+In the configuration directory, the presence of a file ``plugins.json``
+will override the built-in configuration of plugins.  The file format is
+a JSON array containing JSON objects, each of which configures a plugin.
+Example:
+
+.. code-block::
+
+    [
+        {
+            "__bunch__": true,
+            "module": "Info",
+            "tab": "Synopsis",
+            "workspace": "lleft",
+            "start": true,
+            "hidden": true,
+            "category": "System",
+            "menu": "Info [G]",
+            "ptype": "global"
+       },
+       ...
+   ]
+    
+                 
+The keys for each object are defined as follows:
+
+* ``__bunch__``: should be present and set to ``true`` to force
+  deserialization to a ``~ginga.misc.Bunch.Bunch``.
+* ``module``: The name of the module in the ``$PYTHONPATH`` containing
+  the plugin.
+* ``class``: if present, indicates the name of the class within the
+  module that denotes the plugin (if not present the class is assumed
+  to be named identically to the module).
+* ``tab``: the name that the plugin should appear as when opened in a
+  workspace (usually as a tab, but it depends on the type of
+  workspace). Often the same name as the class, but can be different.
+  If not present, defaults to the class or module name of the plugin.
+* ``workspace``: the name of the workspace defined in the layout file
+  (or default layout) where the plugin should be started (see section
+  below on workspace customization).
+* ``start``: ``true`` if the module is of the global type and should
+  be started at program startup. Defaults to ``false``.
+* ``hidden``: ``true`` if the plugin should be hidden from the
```

Comment:
I wonder if `static` is a better name than `hidden`... But maybe that ship has sailed.

Reply:
I don't understand "static" for being left out of the menus, but maybe there is a better keyword. We could consider a rename for a later update but "hidden" has been used for a couple of releases now ... :ship: :smile_cat:

Reply:
Yeah, okay. Let's table this one. Thanks for the clarification!